docs/Systems/LayoutAndSpacing/DemoComponents/SpacingTokens.tsx

```diff
@@ -1,5 +1,5 @@
 import React from "react"
-import { kzSpacing } from "@kaizen/button"
+import { kzSpacing } from "@kaizen/tailwind/src/kz-spacing"
```

Comment:
Weird that you have to dig into the `src`?

Reply:
As discussed, made a new ticket to pull it from `design-tokens` instead.